pkg/k8shandler/desirednodestate.go

```diff
@@ -207,6 +210,35 @@ func (cfg *desiredNodeState) CreateOrUpdateNode(owner metav1.OwnerReference) err
 		if err != nil && !errors.IsAlreadyExists(err) {
 			return fmt.Errorf("Could not create node resource: %v", err)
 		}
+
+		// set it back to being paused
+		cfg.Paused = true
+		nretries := -1
+		if getErr := node.query(); getErr != nil {
+			logrus.Debugf("Could not get Elasticsearch node resource %v: %v", cfg.DeployName, getErr)
+			return getErr
+		}
+		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+			nretries++
+			if getErr := node.query(); getErr != nil {
+				logrus.Debugf("Could not get Elasticsearch node resource %v: %v", cfg.DeployName, getErr)
+				return getErr
+			}
+			dep, updateErr := node.constructNodeResource(cfg, metav1.OwnerReference{})
+			if updateErr != nil {
+				return fmt.Errorf("Could not construct node resource %v for update: %v", cfg.DeployName, updateErr)
+			}
+			logrus.Infof("Updating node resource %v", cfg.DeployName)
```

Comment:
If this is a message that should be printed (as opposed to a debugging message) you should print it only once e.g.
```go
if nretries == 0 {
    logrus.Infof("Updating node resource %v", cfg.DeployName)
}
```
or something like that